### PR TITLE
Display original (not percent-encoded) URLs in HTML and LaTeX output

### PR DESCRIPTION
**Pull request by labdsf**

### CODE REVIEW

File: lib/Text/Amuse/Output.pm [hunk 1]
```diff
@@ -1353,12 +1353,20 @@ sub format_single_link {
             return "\\hyperref{}{amuse}{$linkname}{$linkname}";
         }
     }
+
+    my $url = $self->_url_safe_escape($link);
+    my $desc = $self->manage_regular($link);
```
Comment (melmothx) [inline lib/Text/Amuse/Output.pm]: Here it should use ->safe($link) instead of ->manage_regular to avoid interpolation.

**Comment by labdsf** [comment]
@melmothx I have changed to `safe` and tested attempting to close HTML tag (">") and LaTeX brackets ("}"). No more interpolation, `/*foo*` is not emphasized anymore, as it did with `manage_regular`.

**Comment by melmothx** [comment]
Thanks!

**Comment by melmothx** [comment]
@labdsf Please review fba76ee if you have a chance.

**Pull request merged**